### PR TITLE
♻️ activity: group schedule & execution

### LINKED ISSUE #1261: activity monitor: group timelock proposal and execution

### PR DESCRIPTION
**Pull request by jgalat**
closes #1261 

- Increase events/tx count
- Group executions with schedules.
- Group events and list them if there is no safe TX that contains them

### CODE REVIEW

File: components/RiskFeed/Events/index.tsx [hunk 1]
```diff
@@ -43,65 +49,73 @@ export default function Events({ title, empty, data, calls, isLoading }: Props)
...
+    if (events.length === 0) {
+      continue;
     }
-    if (x.type === 'transaction' && y.type === 'transaction') {
-      return y.data.timestamp - x.data.timestamp;
+
+    const call = events[0];
+    const entry: Entry = { events, timestamp: call.executedAt ?? call.scheduledAt };
+
+    entry.schedule = transactions.find((tx) => tx.timestamp / 1000 === call.scheduledAt);
```
Comment (sebipap) [inline components/RiskFeed/Events/index.tsx:97]: isn't there a need to round the result of the division?

File: components/RiskFeed/Events/index.tsx [hunk 2]
```diff
@@ -359,60 +393,69 @@ function EventSummary({ tx }: EventProps) {
...
+                {format(data.detailedExecutionInfo.executor.value)}
+              </Link>
+            </Value>
+          </Row>
+        )}
+        {isMultiSend ? (
+          <Row title={t('Actions')} inner={inner}>
+            <Box display="flex" flexDirection="column" gap={2}>
+              {data.txData.dataDecoded?.parameters?.[0]?.valueDecoded?.map((v, i) => (
+                <Decode key={i} to={v.to} data={v.dataDecoded} />
```
Comment (sebipap) [inline components/RiskFeed/Events/index.tsx:439]: avoid using array index as `key`. 
https://legacy.reactjs.org/docs/lists-and-keys.html
https://robinpokorny.com/blog/index-as-a-key-is-an-anti-pattern/

**Review by sebipap — DISMISSED** [review]
lgtm! minor comments

File: components/RiskFeed/Events/index.tsx (hunk 1 above)
Comment (jgalat) [reply components/RiskFeed/Events/index.tsx:97]: No, Safe uses `block.timestamp * 1000` for timestamp. All timestamps end with `000`.

**Review by franm91 — APPROVED** [review]

**Pull request merged**